src/tm_devices/components/dm_config_parser.py

```diff
@@ -353,15 +352,22 @@ def __parse_env_options(self) -> DMConfigOptions:
         Raises:
             KeyError: Indicates unrecognized option name.
         """
-        options_list = [
-            arg.strip() for arg in os.getenv(self.OPTIONS_ENV_VARIABLE, "").split(",") if arg
-        ]
+        options_list = {
+            arg.strip().split("=", maxsplit=1)[0]: bool(arg)
+            if "=" not in arg
+            else arg.split("=", maxsplit=1)[-1]
+            for arg in os.getenv(self.OPTIONS_ENV_VARIABLE, "").split(",")
+            if arg
+        }
         valid_config_options = {option.upper() for option in get_type_hints(DMConfigOptions)}
         if valid_config_options.issuperset(options_list):
             options = {
-                arg_name.lower(): arg_name in options_list for arg_name in valid_config_options
+                arg_name.lower(): _convert_config_values(options_list[arg_name])
+                if arg_name in options_list
+                else False
```

Comment:
Should this be `None` if nothing is found?

Reply:
That would make sense to me